hack/gen-crd.sh

```diff
@@ -3,12 +3,12 @@
 set -eu
 
 function annotate_crd() {
-  script='/^metadata:/a\
-\ \ annotations:\
+  script1='/^  annotations:/a\
 \ \ \ \ exclude.release.openshift.io/internal-openshift-hosted: "true"'
```

Comment:
this does not seem related to this PR, please let's drop it.

Reply:
@JoelSpeed going to include the autogen fix for crds in another pr, as I agree it is out of scope